.drone.yml

```diff
@@ -20,37 +20,37 @@ pipeline:
     image: owncloudci/php:${PHP_VERSION}
     pull: true
     commands:
-      - cd /var/www/owncloud/
-      - php occ a:l
-      - php occ a:e oauth2
-      - php occ a:e testing
-      - php occ a:l
-      - php occ config:system:set trusted_domains 1 --value=owncloud
-      - php occ log:manage --level 0
+    - cd /var/www/owncloud/
```

Comment:
Did the fixer do this automagically to this ``yml`` file?

My PHPstorm likes to line up the indent in this way, and I have to fight it. Maybe we should let our editors do this to all ``.drone.yml``

Reply:
I did reformat this file ....